app/graphql/resolvers/rdf.ts

```diff
@@ -242,6 +240,8 @@ export const possibleFilters: NonNullable<
     return [];
   }
 
+  const filters = getFiltersWithSplitIris(_filters);
```

Comment:
Wondering on the naming here.

Wondering if we should have 

dimensionIri = dimension URL without the cube
dimensionId = cubeIRi:dimensionIRI

then the method becomes getFiltersComponentIris.

I think the fact that it's two strings joined is an implementation detail.